grpc-api/src/main/java/io/deephaven/grpc_api/session/TicketRouter.java

```diff
@@ -210,30 +233,30 @@ public void visitFlightInfo(final @Nullable SessionState session, final Consumer
                 .build();
     }
 
-    private TicketResolver getResolver(final byte route) {
+    private TicketResolver getResolver(final byte route, final String logId) {
         final TicketResolver resolver = byteResolverMap.get(route);
         if (resolver == null) {
             throw GrpcUtil.statusRuntimeException(Code.FAILED_PRECONDITION,
-                    "Cannot find resolver for route '" + route + "' (byte)");
+                    "Could not resolve '" + logId + "': no resolver for route '" + route + "' (byte)");
         }
         return resolver;
     }
 
-    private TicketResolver getResolver(final Flight.FlightDescriptor descriptor) {
+    private TicketResolver getResolver(final Flight.FlightDescriptor descriptor, final String logId) {
         if (descriptor.getType() != Flight.FlightDescriptor.DescriptorType.PATH) {
             throw GrpcUtil.statusRuntimeException(Code.FAILED_PRECONDITION,
-                    "Cannot find resolver: flight descriptor is not a path");
+                    "Could not resolve '" + logId + "': flight descriptor is not a path");
         }
         if (descriptor.getPathCount() <= 0) {
             throw GrpcUtil.statusRuntimeException(Code.FAILED_PRECONDITION,
-                    "Cannot find resolver: flight descriptor does not have route path");
+                    "Could not resolve '" + logId + "': flight descriptor does not have route path");
         }
 
         final String route = descriptor.getPath(0);
         final TicketResolver resolver = descriptorResolverMap.get(route);
         if (resolver == null) {
             throw GrpcUtil.statusRuntimeException(Code.FAILED_PRECONDITION,
```

Comment:
any chance you want these to be NOT_FOUND instead? FAILED_PRECONDITION isn't inaccurate, but NOT_FOUND could make more sense with nonsense tickets, potentially

Reply:
I feel that NOT_FOUND implies that the ticket is structured correctly, but that it just doesn't exist. I've tried to use FAILED_PRECONDITION if the ticket is messed up or unparseable somehow.